app/helpers/green_lanes_helper.rb

```diff
@@ -31,15 +31,41 @@ def render_exemptions_or_no_card(category, assessments)
   end
 
   def exemptions_met?(category, category_assessment, answers)
+    return false if answers.nil?
+
     category = category.to_s
 
     category_assessment_answer = answers.dig(category, category_assessment.category_assessment_id.to_s)
 
     category_assessment_answer.present? && category_assessment_answer != %w[none]
   end
 
+  def all_exemptions_met?(category, category_assessments, answers)
+    category_assessments.all? { |ca| exemptions_met?(category, ca, answers) }
+  end
+
+  def render_exemptions(assessments, category)
+    if category.to_s == '3'
+      render_category_3_exemptions(assessments) if assessments.cat_1_exemptions_met || assessments.cat_2_exemptions_met
+    else
+      render_other_category_exemptions(assessments)
+    end
+  end
+
   private
 
+  def render_category_3_exemptions(assessments)
+    safe_join([render_exemptions_or_no_card(1, assessments), render_exemptions_or_no_card(2, assessments)])
+  end
+
+  def render_other_category_exemptions(assessments)
```

Comment:
this is called only my `render_exemptions` if I am not wrong

Should we keep the logic inside render_exemptions?
It'd be easier to test (and we must add tests for it)

Reply:
I moved out to make easy of breaking it down aswell